frontend/src/components/CardReport.tsx

```diff
@@ -1,49 +1,42 @@
-import { v4 as uuidv4 } from 'uuid'
+import { Card, CardDescription, CardFooter, CardHeader, CardTitle } from "./Card";
+import { Button } from "./Button";
+import heartOutline from "../assets/icons/heartOutline.icon.svg"
```

Comment:
Dùng "~/" để tìm thư mục chứ không dùng ../ sau này khó refact, sửa lại cho tất cả chỗ mà m dùng ../
```suggestion
import heartOutline from '~/assets/icons/heartOutline.icon.svg'
```